change/@ni-nimble-angular-cae1f20e-b472-42ea-9f72-309d57cb3171.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Comment:
I was about to say this should be `major` since we're removing a public API, but the table is still in "incubating" so I think this can be `patch`